spirv_cross/src/hlsl.rs

```diff
@@ -7,6 +7,10 @@ use std::marker::PhantomData;
 #[derive(Debug, Clone)]
 pub struct Target;
```

Comment:
out of curiosity, why not `enum Target {}` like my changes did?

Reply:
I was playing with the idea of storing target data directly on `Target` but forgot to revert it back to `enum` afterwards :smiley:. Is there an advantage of `enum Target {}` over `struct Target`?

Reply:
Not much, just the type level enforcement to not being able to instance one, so it's a little bit more idiomatic

Reply:
Got it, thanks!